src/Makefile.am

```diff
@@ -283,12 +283,12 @@ BITCOIN_CORE_H = \
   uint256.h \
   uint252.h \
   undo.h \
-  util.h \
+  util/system.h \
   util/match.h \
-  utilmoneystr.h \
-  utilstrencodings.h \
+  util/moneystr.h \
+  util/strencodings.h \
   utiltest.h \
```

Comment:
Was `utiltest.h` introduced by us, or by upstream in a different PR, or? Doesn't necessarily need to be done in this PR, but might be nice as an additional commit - if it was introduced by us. If it was from upstream, we should figure out why it wasn't handled in the PR being backported here.